src/main/charts/confluence/templates/clusterrole.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.serviceAccount.clusterRole.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "confluence.clusterRoleName" . }}
+  labels:
+  {{- include "confluence.labels" . | nindent 4 }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - endpoints
+      - pods
+      - nodes
+    verbs:
+      - get
+      - list
+{{- end -}}
```

Comment:
This files defines the permissions that the hazelcast-kubernetes adapter needs to perform peer discovery